src/common/MapLayer.ts

```diff
@@ -18,6 +19,7 @@ export class MapLayer extends LayerGroup {
     private map: ZDMap,
     public layerName: string,
     tilePath: string | undefined,
+    enabledTilePath: string | undefined,
```

Comment:
"enabledTilePath" is confusing because we aren't actually changing tiles. Let's either:
1. Continue to ~be lazy~ favor convention over configuration and just hardcode "${tilePath}-selected.png"
2. Separate into three parameters: "tilePath", "iconPath", and "enabledIconPath"

I'm fine with either.

Reply:
or:
3. rename `tilePath` to `layerId` and hardcode "${layerId}-selected.png"
4. rename `tilePath` to `layerId` and `enabledTilePath` to `layerIdEnabled`

Reply:
Chose 4th option as to avoid any complications by having to check existence of the layerIdEnabled icon, or being lazy and not making a separate icon for the enabled state resulting in undefined.